src/errors.test.ts

```diff
@@ -18,17 +19,6 @@ describe("errors.ts logError()", () => {
     sandbox.restore();
   });
 
-  const createResponseError = (status: number, statusText: string, body: string): ResponseError => {
```

Comment:
Hoisted up to become `testUtils.createResponseError()` so can be used by other suites.